crates/db_views_actor/src/community_moderator_view.rs

```diff
@@ -27,6 +27,20 @@ impl CommunityModeratorView {
     .get_result::<bool>(conn)
     .await
   }
+
+  pub async fn is_community_moderator_of_any(
+    pool: &mut DbPool<'_>,
+    find_person_id: PersonId,
+  ) -> Result<bool, Error> {
+    use lemmy_db_schema::schema::community_moderator::dsl::{community_moderator, person_id};
+    let conn = &mut get_conn(pool).await?;
+    select(exists(
+      community_moderator.filter(person_id.eq(find_person_id)),
+    ))
+    .get_result::<bool>(conn)
+    .await
+  }
```

Comment:
This is only used in `is_mod_of_any_or_admin()` so you can merge it into that.

Reply:
I'd rather not merge it into that, because that one also checks for admins. I'll mark pub crate tho.